src/tests/_suite.ts

```diff
@@ -49,7 +50,7 @@ describe('azure pipeline task tests', function() {
       'uri',
```

Comment:
just noticing the test names still mention azure pipelines(this was originally adapted from our azure pipelines plugin).  Wondering if you mind cleaning that up while you're in there?

Reply:
Sure. Done.